### ISSUE TITLE
Connection to SSH agent refused

### ISSUE THREAD

**Issue by amirmohsen** [cmderdev/cmder#1123]
I started cmder this morning and got a notification to update. So I did and now I can't seem to use ssh anymore. Here's what happens:

```
$  eval `ssh-agent`
Agent pid [NUMBER]

$ ssh-add
Error connecting to agent: Connection refused
```

I use Git bash (Mingw64) with cmder. I tried Git bash directly and SSH works fine over there so it's probably not a problem with that.

I am on Windows 10 64bit and running ConEmu version 161009 stable. My git bash is mintty 2.5.0 (x86_pc-msys).

This was working perfectly fine before this morning's update.

**Comment by amirmohsen** [comment]
I've downgraded to 160710 now and the ssh agent works again.

**Comment by PieterScheffers** [comment]
I have the same issue. 

If I execute `"C:\Program Files\Git\cmd\start-ssh-agent.cmd"`. I get the 'connection refused' message.

If I first start the ssh-agent with `ssh-agent -s` in a Git Bash console, and then run `"C:\Program Files\Git\cmd\start-ssh-agent.cmd"`, it works.

**Comment by PieterScheffers** [comment]
I also downgraded, but to the 161002 version. Now it works again.

**Comment by MartiUK** [comment]
Looks like there's a problem with 161009 and git, stay on 161002 for the time being.

**Comment by au** [comment]
Running `start-ssh-agent` works just fine on my end.

![ssh](https://cloud.githubusercontent.com/assets/10539834/19244426/e8294604-8ee1-11e6-86f7-aaf89c6aadb7.PNG)

**Comment by Stanzilla** [comment]
Fixed in 161009a
